Bottom: 9dae5c54cda195cd3d3fe6d457f4b286139c7509
Top:    9a6d86e31bf0735d99f93466c621a9712df44108
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-06 12:54:55 +0800

Refresh of autofs-5.0.5-fix-master-map-bogus-keywork-match.patch

---

diff --git a/lib/master_parse.y b/lib/master_parse.y
index 00b7935..680f116 100644
--- a/lib/master_parse.y
+++ b/lib/master_parse.y
@@ -541,11 +541,11 @@ options: option {}
 		master_notify($1);
 		YYABORT;
 	}
-	/*| options EQUAL
+	| options EQUAL
 	{
 		master_notify($1);
 		YYABORT;
-	}*/
+	}
 	;
 
 option: daemon_option
